### PR TITLE
[fix bug] nil StatusCode handling on HTTP Client & Iris error type field struct

### PR DESCRIPTION
**Pull request by Xaxxis**
- Fix nil handling, when HTTP timeout
- Update value bank account for test
- Fix field error type on iris CreatePayoutResponse struct

### CODE REVIEW

File: httpclient.go [hunk 1]
```diff
@@ -97,66 +97,78 @@ func (c *HttpClientImplementation) DoRequest(req *http.Request, result interface
 	res, err := c.HttpClient.Do(req)
 	if err != nil {
 		c.Logger.Error("Cannot send request: %v", err.Error())
+		var statusCode int
+
+		if res != nil {
```
Comment (rizdaprasetya) [inline httpclient.go:102]: when `res != nil` is there any chance that `res.StatusCode` nil? maybe add additional `&& res.StatusCode != nil` to be safe?

**Review by rizdaprasetya — COMMENTED** [review]
Overall looks good thanks! 👍  just one comment.

File: iris/response.go [hunk 2]
```diff
@@ -29,7 +29,7 @@ type BeneficiaryBankResponse struct {
 type CreatePayoutResponse struct {
 	Payouts      []CreatePayoutDetailResponse `json:"payouts"`
 	ErrorMessage string                       `json:"error_message"`
-	Errors       []string                     `json:"errors"`
+	Errors       interface{}                  `json:"errors"`
```
Comment (rizdaprasetya) [inline iris/response.go:32]: it probably good idea to add test-case for this scenario too, to make sure no error is thrown, when this struct is called

File: httpclient.go (hunk 1 above)
Comment (Xaxxis) [reply httpclient.go:102]: > when `res != nil` is there any chance that `res.StatusCode` nil?

Nope, the value status code will `0`. So it is safe to check nil only on res object `res != nil`

File: iris/response.go (hunk 2 above)
Comment (Xaxxis) [reply iris/response.go:32]: Sure, will create the test for this one

Comment (Xaxxis) [reply iris/response.go:32]: Done, added the CreatePayoutFail test

**Review by rizdaprasetya — APPROVED** [review]
Looks good now thanks 👍  please go ahead merge.
Another note: there seems to be no instruction or readme file on example folder, can you create instructions on how to run the examples @Xaxxis ? thanks

**Pull request merged**